.github/workflows/gke-k8s-chart-support-e2e.yaml

```diff
@@ -0,0 +1,62 @@
+# This workflow calls the main workflow with custom variables
+name: GKE-K8s-Chart-Support-E2E
+
+on:
+  workflow_dispatch: # Allow running manually on demand
+    inputs:
+      rancher_version:
+        description: Rancher version to deploy
+        required: true
+        type: string
+        default: 2.8.2
+      k3s_version:
+        description: k3s version of local cluster
+        required: true
+        type: string
+        default: v1.27.9+k3s1
+      run_k8s_chart_support_provisioning_tests:
+        required: true
+        default: false
+        type: boolean
+      run_k8s_chart_support_importing_tests:
+        required: true
+        default: false
+        type: boolean
+      destroy_runner:
+        description: Destroy the auto-generated self-hosted runner
+        default: true
+        type: boolean
+      downstream_cluster_cleanup:
+        description: Cleanup downstream clusters after test
+        default: true
+        type: boolean
+      runner_template:
+        description: Runner template to use
+        default: hosted-prov-e2e-ci-runner-spot-n2-highmem-16-gl-template-v1
+        type: string
+      k8s_upgrade_minor_version:
```

Comment:
```suggestion
      k8s_version_to_test:
```

Reply:
Previously it was `k8s_version_to_test` but that was slightly confusing since it indicates that you might have to provide a complete version, but that is not true. We only need to pass the minor version, for e.g. 1.28, the complete version will be computed programmatically.

Reply:
_k8s_minor_version_to_test_ ?

Reply:
I have removed the GH action for now. I will add it once AKS and EKS is implemented.